src/main/java/org/mskcc/cbio/oncokb/web/rest/UsageAnalysisController.java

```diff
@@ -0,0 +1,140 @@
+import org.springframework.http.HttpStatus;
+import org.mskcc.cbio.oncokb.config.application.ApplicationProperties;
+import org.mskcc.cbio.oncokb.web.rest.vm.usageAnalysis.UsageSummary;
+import org.mskcc.cbio.oncokb.web.rest.vm.usageAnalysis.UserOverviewUsage;
+import org.mskcc.cbio.oncokb.web.rest.vm.usageAnalysis.UserUsage;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/api")
+public class UsageAnalysisController {
+
+    @Autowired
+    private ApplicationProperties applicationProperties;
+
+    private String s3AccessKey = "";
+    private String s3SecretKey = "";
+    private String s3Region = "";
+
+    final String ONCOKB_BUCKET = "oncokb";
+    final String USERS_USAGE_SUMMARY_FILE = "usage-analysis/userSummary.json";
+    final String RESOURCES_USAGE_SUMMARY_FILE = "usage-analysis/resourceSummary.json";
+
+    @PostConstruct
+    private void init() {
+        s3AccessKey = applicationProperties.getAws().getS3AccessKey();
+        s3SecretKey = applicationProperties.getAws().getS3SecretKey();
+        s3Region = applicationProperties.getAws().getS3Region();
+    }
+
+    private JSONObject requestData(String bucket, String file)
+            throws UnsupportedEncodingException, IOException, ParseException {
+        AWSCredentials credentials = new BasicAWSCredentials(s3AccessKey, s3SecretKey);
+        AmazonS3 s3client = AmazonS3ClientBuilder.standard()
+                .withCredentials(new AWSStaticCredentialsProvider(credentials)).withRegion(s3Region).build();
+
+        S3Object s3object = s3client.getObject(bucket, file);
+        S3ObjectInputStream inputStream = s3object.getObjectContent();
+        JSONParser jsonParser = new JSONParser();
+        return (JSONObject) jsonParser.parse(new InputStreamReader(inputStream, "UTF-8"));
+    }
+
+    @GetMapping("/usage/user/{userId}")
+    public ResponseEntity<UserUsage> userUsageGet(@PathVariable String userId)
+        throws IOException, ParseException {
+    
+        HttpStatus status = HttpStatus.OK;
+
+        JSONObject jsonObject = requestData(ONCOKB_BUCKET, USERS_USAGE_SUMMARY_FILE);
+
+        if (jsonObject.containsKey(userId)){
+            JSONObject usageObject = (JSONObject)jsonObject.get(userId);
+            Gson gson = new Gson();
+            UserUsage userUsage = gson.fromJson(usageObject.toString(), UserUsage.class);
+            return new ResponseEntity<UserUsage>(userUsage, status);
+        }
+
+        return new ResponseEntity<UserUsage>(new UserUsage(), status);
+        
+    }
+
+    @GetMapping("/usage/user/overview")
```

Comment:
I would rather use "/usage/overview". I would not mess with the method `"/usage/users/{userId}"